_projects/SP19-Fake News.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: project
+title: Fake News
+image: <link to project image (optional)>
+desc: |
+ Slice fake news articles for fun and for profit.
+github: <github link (optional)>
+dl: https://drive.google.com/open?id=1HlgbFNRmI1L1uYKqxMwQ3baDfqM6HIMx
+semester: Spring 2019
+team: 25
+youtube: lp4a7KxF64g
```

Comment:
Video is unavailable

Reply:
Weird the video is working for me (https://www.youtube.com/watch?v=lp4a7KxF64g), what's showing up for you?

Reply:
It showed up as unavailable, but it seems to work now